### ISSUE TITLE
Automatic support providing answers

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#17]
## User story
1. As a IT support team member
2. I want / need an automatic support answering tickets 
3. So that I can focus on the more complicated tickets

## Acceptance criteria
- [ ] the answer is to be created without manual effort
- [ ] I as an ticket agent am informed that the automatic support will take care of certain tickets
- [ ] I am informed when an answer has been generated before it is sent
- [ ] the answer states a feasible solution to the reported problem 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by AnnaH3003** [comment]
If training the AI and getting the certainty score is independent of each other, it should be done after another
